pylint/pyreverse/main.py

```diff
@@ -56,15 +56,11 @@
             "type": "string",
             "action": "store",
             "metavar": "<mode>",
-            "help": """filter attributes and functions according to
-    <mode>. Correct modes are :
-                            'PUB_ONLY' filter all non public attributes
-                                [DEFAULT], equivalent to PRIVATE+SPECIAL_A
-                            'ALL' no filter
-                            'SPECIAL' filter Python special functions
-                                except constructor
-                            'OTHER' filter protected and private
-                                attributes""",
+            "help": "filter attributes and functions according to <mode>. Correct modes are:"
+            "'PUB_ONLY' - filter all non public attributes;"
```

Comment:
`PUB_ONLY` is the default, and thus we should not remove this information from the help message.